modules/core/map/src/Element/FarmMap.php

```diff
@@ -40,8 +40,8 @@ public function getInfo() {
    *   appended to 'farm-map-' as the map element ID if one has not already
    *   been provided.
    *
-   * @return \Drupal\farm_map\Element\FarmMap
-   *   The farm map render element.
+   * @return array
+   *   The farm map render element build array.
```

Comment:
I think we should maybe change this to `FarmMap|array`. We return the value of `$event->element` which is typed as either: https://github.com/farmOS/farmOS/blob/0157bf49b1b6d56f76861ceed5a7f56929beec2a/modules/core/map/src/Event/MapRenderEvent.php#L20-L25

I think there is something weird behind the scenes here where it ends up as either an array or the actual render element. I never fully figured it out which is why I kept both types... Probably not a biggies because I don't think these type doc changes would break strict type rules, but maybe worth looking into again.